### PR TITLE
indent standardized to 2 spaces

### PR DESCRIPTION
**Pull request by AlexeyNikipelau**

### CODE REVIEW

File: src/base/Components.ts [hunk 1]
```diff
@@ -1,108 +1,109 @@
 /**
- * Manages all of the components within the Form.io renderer.
- */
+* Manages all of the components within the Form.io renderer.
```
Comment (travist) [inline src/base/Components.ts:2]: Seemed to mess up the comment spacing..... It should be
```
/**
 *
 */
```

Comment (travist) [reply src/base/Components.ts:2]: Or is my assumption wrong about block comments?

Comment (AlexeyNikipelau) [reply src/base/Components.ts:2]: that was done automatically by vs code, we can change it if needed

**Review by travist — CHANGES_REQUESTED** [review]
Please fix block comments.

**Comment by AlexeyNikipelau** [comment]
Agreed that indent should be 4 spaces, closing the pr, will be replaced with new one making 4 spaces indent

**Pull request closed without merging**